packages/compose/src/index.native.js

```diff
@@ -0,0 +1,9 @@
+
+// Web exports
+export * from './index.js';
```

Comment:
This one looks hacky but we can address it later by introducing two new files:

- `packages/compose/src/higher-order/index.js`
- `packages/compose/src/hooks/index.js`